privateSBOMExchange/src/petra/lib/models/__init__.py

```diff
@@ -626,7 +678,11 @@ def visit_field_node(self, node: FieldNode):
         """
         if node.encrypted_data != NODE_PUBLIC:
             try:
-                node.decrypted_data = bytes(cpabe_decrypt(self.secret_key, node.encrypted_data))
+                node.decrypted_data = decrypt_data_AES(node.encrypted_data, self.__decrypted_aes_keys[node.policy])
+                #node.decrypted_data = bytes(cpabe_decrypt(self.secret_key, node.encrypted_data))
```

Comment:
do we need this line?

Reply:
I had left these in here _just in case_ we wanted to be able to switch, but I'll remove these